dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -410,7 +411,7 @@
       <PRIResource Remove="@(PRIResource)" />
     </ItemGroup>
     <!-- Use CreateItem to expand all the ** to individual full paths -->
-    <CreateItem Include="@(PRIResourceExpanded -> '%(Replaced)')">
+    <CreateItem Include="@(PRIResourceExpanded -&gt; '%(Replaced)')">
```

Comment:
>-&gt;  [](start = 47, length = 6)

I don't expect this change become '->' is transform modifiers in msbuild. see [https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-transforms?view=vs-2015](https://docs.microsoft.com/en-us/visualstudio/msbuild/msbuild-transforms?view=vs-2015)

Reply:
The CreateNewProject script is the one doing this, because it writes the project file back in proper xml. It is harmless.